core/chains/evm/txmgr/stuck_tx_detector.go

```diff
@@ -363,13 +364,6 @@ func (d *stuckTxDetector) SetPurgeBlockNum(fromAddress common.Address, blockNum
 }
 
 func (d *stuckTxDetector) StuckTxFatalError() *string {
```

Comment:
Is there still value in having a method for this if there is no logic?

Reply:
I believe so just because this method is called from the generalized Confirmer code [here](https://github.com/smartcontractkit/chainlink/blob/796073a82d8044d39e51b2bc97b4790bf6cc1124/common/txmgr/confirmer.go#L567) which needs to be served the chain-specific error message for a terminally stuck tx. We could remove this once the Confirmer is degeneralized though.